src/mococrw/distinguished_name.h

```diff
@@ -245,4 +255,16 @@ DistinguishedName::Builder &DistinguishedName::Builder::title(T &&name)
     return *this;
 }
 
+/**
+ * Sets the initial attribute in the distinguished name builder
```

Comment:
I agree, especially since the other functions are also undocumented. I raised this inconsistency, however I was explicitly asked to add the comments for the doxygen. It is also part of the acceptance criteria.